app/icons/icons.qrc

```diff
@@ -87,5 +87,6 @@
         <file>XTwitter.svg</file>
         <file>Youtube.svg</file>
         <file>ZoomToProject.svg</file>
+        <file>GpsDirection.svg</file>
```

Comment:
revert you do not need it. This is always composed from 
1/ direction image (triangle where you look)
2/ marker (dot in middle)
3/ light circle (dynamic - that shows your GPS accuracy - e.g. if you GPS has 10 cm accuracy your green circle is 10cm wide)